### ISSUE TITLE
Reproducible code needed

### ISSUE THREAD

**Issue by Jieun316** [suduo94/D4M#14]
Hi, Thank you for your nice work.

I just wanna reproduce the distilled results on CIFAR datasets, so could you upload your pre-trained model or implementation details for CIFAR10/100 such as batch size, learning rate and the number of epochs, etc. The code only contains hyper-parameters for ImageNet dataset. 
When I was trying to train from scratch for CIFAR10, I only achieved accuracy of 23%. How can I achieve accuracy of 59% as represented in the paper?
